execution/block-partitioner/src/sharded_block_partitioner/config.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright © Aptos Foundation
+
+use crate::{sharded_block_partitioner::ShardedBlockPartitioner, BlockPartitioner};
+use aptos_types::block_executor::partitioner::RoundId;
+
+#[derive(Clone, Copy, Debug)]
+pub struct PartitionerV1Config {
```

Comment:
Follow up on the comment regarding the v1, v2 config hierarchies.
We have a file called `../../config.rs`, but it has only  `PartitionerV1Config`

Reply:
Same reason: `sharded_block_partitioner`=`v1`...